examples/android-instrumentation-sample/src/main/java/io/sentry/samples/instrumentation/ui/MainActivity.kt

```diff
@@ -24,14 +28,21 @@ class MainActivity : ComponentActivity() {
         list.adapter = TrackAdapter()
 
         lifecycleScope.launchWhenStarted {
+            Sentry.getSpan()?.finish()
+            val transaction = Sentry.startTransaction(
+                "Track Interaction",
+                "ui.action.load",
+                true
+            )
```

Comment:
If the `collect` does not get called because of no items, the transaction will never be finished, or does it get called anyway?

Reply:
mm, I think it will always emit the latest value + we prefill the db from file, so it won't ever be empty. Should be fine I think